lib/common/presentation/views/faq/faq_screen.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+
+/// The page [FaqScreen]
+class FaqScreen extends StatelessWidget {
+  /// Constructs an instance of [FaqScreen]
+  const FaqScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Padding(
+        padding: const EdgeInsets.all(40),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Row(
+              children: [
+                Image.asset(
+                  'assets/images/speechbubbleimage.png',
+                  height: 100,
+                  width: 100,
+                ),
+                const SizedBox(width: 20),
+                const Text(
+                  'FAQ Page',
+                  style: TextStyle(fontSize: 40, fontWeight: FontWeight.bold),
```

Comment:
Use the default defined theme, don't define a custom textstyle.

with vanilla flutter, access the theme via `Theme.of(context)...`